app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -117,3 +120,14 @@ examples:
         hint_text: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus sapien justo, lobortis elementum tortor in, luctus interdum turpis. Nam sit amet nulla nec arcu condimentum dapibus quis varius metus. Suspendisse cursus tristique diam et vestibulum. Proin nec lacinia tortor. Morbi at nisi id lorem aliquam ullamcorper. Pellentesque laoreet sit amet leo sodales ultricies. Suspendisse maximus efficitur odio in tristique."
         text: "Quisque tincidunt venenatis bibendum. Morbi volutpat magna euismod ipsum consequat cursus. Etiam bibendum interdum ultricies."
         bold: true
+  conditional:
```

Comment:
This is a nice approach, but I wonder if we need to change something to support revealing other elements than radio buttons. I'm specifically thinking about the  [review reminder](https://whitehall-prototype.herokuapp.com/content-settings), which reveals a block with other form things:

<img width="499" alt="screen shot 2018-07-18 at 08 03 14" src="https://user-images.githubusercontent.com/233676/42865296-21182f8a-8a61-11e8-824a-f65dd29d4d0a.png">

Or on the public site, the [contact GOV.UK form](https://www.gov.uk/contact/govuk), which would reveals a string input:

<img width="805" alt="screen shot 2018-07-18 at 08 02 05" src="https://user-images.githubusercontent.com/233676/42865214-e1f10318-8a60-11e8-922c-6a529f18e609.png">

Perhaps we can pass in the conditional HTML to the option?

Reply:
I think I completely misread the PR at first, it already does what I suggested.. Never mind me!